bounties/rubygems/webbynode/1/vulnerability.json

```diff
@@ -0,0 +1,49 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-08-04",
+	"AffectedVersionRange": "*",
+	"Summary": "Arbitrary Command Execution",
+	"Author": {
+		"Username": "",
+		"Name": "Unknown"
+	},
+	"Package": {
+		"Registry": "rubygems",
+		"Name": "webbynode",
+		"URL": "https://rubygems.org/gems/webbynode",
+		"Downloads": "69300"
+	},
+	"CWEs": [{
+		"ID": "CWE-77",
+		"Description": ""
+	}],
+	"CVSS": {
+		"Version": "3.1",
+		"AV": "N",
+		"AC": "L",
+		"PR": "N",
+		"UI": "N",
+		"S": "U",
+		"C": "L",
+		"I": "L",
+		"A": "L",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "7.3"
+	},
+	"CVEs": [
+		"CVE-2013-7086"
+	],
+	"Repository": {
+		"URL": "http://github.com/webbynode/webbynode",
+		"Codebase": [
+			"Ruby"
+		]
+	},
+	"Permalinks": [],
+	"References": [{
+		"Description": "rubysec.com",
+		"URL": "http://rubysec.com/advisories/CVE-2013-7086"
```

Comment:
Returns a `404` error. Please replace the expired link with https://www.cvedetails.com/cve/CVE-2013-7086/ :smile: